corehq/tests/noseplugins/dividedwerun.py

```diff
@@ -84,7 +84,7 @@ def skip():
                         skip.__name__ = "*"
                     skip.__module__ = name
             else:
-                desc = test.test
+                desc = test.test.descriptor
```

Comment:
I wonder if this is necessary in all cases. Wonder if it might be more correct to do:
```py
if hasattr(test.test, "__name__"):
    desc = test.test
else:
    desc = test.test.descriptor
```

Shame on me for not writing tests for this 😞